SPECS/podman/podman.spec

```diff
@@ -0,0 +1,3191 @@
+%global with_check 0
+%global with_debug 1
```

Comment:
Since it's set to a constant and builds, let's just remove `with_debug` completely and keep the bits, which were enabled now.

Reply:
Yup!